bag_ecd/bag_design.py

```diff
@@ -146,7 +145,9 @@ class of this package.
         
         elif os.path.isfile(schematic_generator):
             print('Schematic generator exists at %s. Trying to map that to generated one.' %(schematic_generator))
-            
+            print('Copying schematic generator to %s ' %(packagename))
+            # This needs to be done, as new_package is now True, meaning that there is file at packagename!
```

Comment:
Actually, this should be "This needs to be done, as new_package is now True, meaning that there is NO file at packagename!".

Reply:
I think this can be reverted as we have debugged the reason to be somewhere else.